### ISSUE TITLE
MAC: "Your GStreamer installation is missing a plug-in."

### ISSUE THREAD

**Issue by winwisely99** [pion/rtwatch#14]
brew install gst-plugins-good pkg-config

````
export PKG_CONFIG_PATH="/usr/local/opt/libffi/lib/pkgconfig"
go run main.go -container-path=/Users/apple/workspace/go/src/github.com/winwisely99/plugins/video/WebRTC/watch/video.mp4
go: finding github.com/pion/rtwatch latest
Video file '/Users/apple/workspace/go/src/github.com/winwisely99/plugins/video/WebRTC/watch/video.mp4' is now available on ':8080', have fun!
GStreamer Error: Your GStreamer installation is missing a plug-in.
exit status 1
make: *** [go-run] Error 1
````

main.go is the same as this repo

**Comment by Sean-Der** [comment]
Hey @winwisely99 

That is a mistake with the docs, can you install `gst-plugins-bad gst-plugins-ugly` and try again. If that works I will update the docs

thanks!

**Comment by Sean-Der** [comment]
Fixed by d39989b3ffa9e17319b0d39228c3a722d5acb222
